### DISCUSSION TITLE
IPyaladin widget not working in VSCode

### DISCUSSION THREAD

**Discussion by JosephKarpinski** [microsoft/vscode-jupyter#9290] [Questions and Answers] [answered]
Hi, 

I’m lost on this.
This jupyter notebook works fine in Jupyter but not in Visual Studio.
Nothing is displayed. Attaching error message.

<img width="526" alt="D1985D1C-B788-4F0D-862B-4CC70347281D" src="https://user-images.githubusercontent.com/37123616/157284667-2251356a-10af-4f91-82f0-f764ca382fca.png">
<img width="918" alt="55C0516D-26D4-4594-B4D9-DE69CB479B6D" src="https://user-images.githubusercontent.com/37123616/157284669-0420abbd-e542-482e-b733-93e37cd56de2.png">
<img width="526" alt="21E98D11-AF81-4277-92AA-AC6078FF55ED" src="https://user-images.githubusercontent.com/37123616/157284671-fadf5af2-977a-44ab-ac5f-ec25a1cc5f25.png">
<img width="253" alt="3ABC4FB5-B3D4-41F0-A18D-222B20161A3D" src="https://user-images.githubusercontent.com/37123616/157284672-8e9ebea1-2b01-4289-8cda-e779c52eb723.png">
<img width="1091" alt="6EA74317-BC6A-45DC-A4F9-BC831910A28A" src="https://user-images.githubusercontent.com/37123616/157284674-0c9058da-312f-4051-ad8b-02fe58b98bc1.png">

**Comment by DonJayamanne** [comment]
Thanks for filing this this, it looks like ipyaladin widget does not work in VS Code, as that's a bug I'm transferring this discussion to an issue https://github.com/microsoft/vscode-jupyter/issues/9292.

**Reply by rchiodo** [reply]
To me it looks like the onus here is on ipyaladin. Most widgets publish their js files on npm or other locations. According to jsdelivr.com, ipyaladin does not have anything published:

![image](https://user-images.githubusercontent.com/19672699/157346023-0c43431c-f08f-454a-89c9-b2a8c13959e0.png)

**Comment by rchiodo** [comment]
See the wiki here:
https://github.com/microsoft/vscode-jupyter/wiki/IPyWidget-Support-in-VS-Code-Python

**Comment by DonJayamanne** [comment, marked as answer]
This is a bug in the Jupyter extension and the issue is tracked here https://github.com/microsoft/vscode-jupyter/issues/9292
It will probably be fixed in the next release.
